src/index.html

```diff
@@ -3,11 +3,52 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="star"></div>
+      <div class="star"></div>
+      <div class="star"></div>
+      <div class="star"></div>
+      <div class="star"></div>
+    </div>
+    <div class="stars stars--1">
+      <div class="active-star"></div>
+      <div class="star"></div>
+      <div class="star"></div>
+      <div class="star"></div>
+      <div class="star"></div>
+    </div>
+    <div class="stars stars--2">
+      <div class="active-star"></div>
+      <div class="active-star"></div>
+      <div class="star"></div>
+      <div class="star"></div>
+      <div class="star"></div>
+    </div>
+    <div class="stars stars--3">
+      <div class="active-star"></div>
+      <div class="active-star"></div>
+      <div class="active-star"></div>
+      <div class="star"></div>
+      <div class="star"></div>
+    </div>
+    <div class="stars stars--4">
+      <div class="active-star"></div>
+      <div class="active-star"></div>
+      <div class="active-star"></div>
+      <div class="active-star"></div>
+      <div class="star"></div>
+    </div>
+    <div class="stars stars--5">
```

Comment:
```suggestion
      <div class="active-star"></div>
      <div class="star"></div>
      <div class="star"></div>
      <div class="star"></div>
      <div class="star"></div>
    </div>
    <div class="stars stars--2">
      <div class="active-star"></div>
      <div class="active-star"></div>
      <div class="star"></div>
      <div class="star"></div>
      <div class="star"></div>
    </div>
    <div class="stars stars--3">
      <div class="active-star"></div>
      <div class="active-star"></div>
      <div class="active-star"></div>
      <div class="star"></div>
      <div class="star"></div>
    </div>
    <div class="stars stars--4">
      <div class="star"></div>
      <div class="star"></div>
      <div class="star"></div>
      <div class="star"></div>
      <div class="star"></div>
    </div>
    <div class="stars stars--5">
```
блок star нічого не знає про те активний він чи ні, це йому диктувати має контейнер, а стилі встановлювати відповідной background, зараз це хардкод